### DISCUSSION TITLE
add Myst node to a balena software nebra

### DISCUSSION THREAD

**Discussion by Ondevento** [community/community#57515] [Pull Requests]
### Select Topic Area

Question

### Body

i have a controllino that works on helium with nebra software on balena dashboard. How can I install https://github.com/otkd/balena-myst doing dual mining?

**Comment by otter-computer** [comment]
Hi there,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the project's repository. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
